tests/tests.csv

```diff
@@ -98,12 +99,12 @@ rt;/a?b#c;[true, ["a"], ["b"], "c"];;coaps://foo:4711/a?b#c;[-2, ["foo", 4711],
 rt;?a&b&c;[0, null, ["a", "b", "c"]];;coaps://foo:4711/pa/th?a&b&c;[-2, ["foo", 4711], ["pa", "th"], ["a", "b", "c"]];8300f683616161626163;84218263666f6f1912678262706162746883616161626163;
 rt;;[];;coaps://foo:4711/pa/th?query#frag;[-2, ["foo", 4711], ["pa", "th"], ["query"], "frag"];80;85218263666f6f19126782627061627468816571756572796466726167;Empty CRI Reference ~ [0]
 rt;//a.b;[null, ["a", "b"]];;coaps://a.b;[-2, ["a", "b"]];82f68261616162;82218261616162;Dotted host name
-rt;//a%2Ea;[null, ["a.a"]];;coaps://a%2Eb;[-2, ["a.a"]];82f68163612e61;82218163612e61;Weird non-PET in host
+rt;//a%2Ea;[null, ["a.a"]];;coaps://a%2Eb;[-2, ["a.a"]];82f68163612e61;82218163612e61;Weird non-PET in host (not allowed: URI normalization would turn that into coaps://a.b);broken
```

Comment:
Right now the information that this is broken is not written down as part of the constraints but only as part of the conversion rules; C5 should say that.